https://bugs.gentoo.org/show_bug.cgi?id=167052 attachment #126176

Fix multi-domain systems including sparc and pegasos ppc

diff -Nura xorg-server-1.3.0.0/hw/xfree86/os-support/bus/linuxPci.c xorg-server-1.3.0.0-nosysfs/hw/xfree86/os-support/bus/linuxPci.c
--- xorg-server-1.3.0.0/hw/xfree86/os-support/bus/linuxPci.c	2006-11-16 15:01:25.000000000 -0300
+++ xorg-server-1.3.0.0-nosysfs/hw/xfree86/os-support/bus/linuxPci.c	2007-07-27 13:48:58.000000000 -0300
@@ -148,7 +148,7 @@
 	int		domain, bus, dev, func;
 	char		file[64];
 	struct stat	ignored;
-	static int is26 = -1;
+	static int is26 = 0;
 
 	domain = PCI_DOM_FROM_TAG(tag);
 	bus  = PCI_BUS_NO_DOMAIN(PCI_BUS_FROM_TAG(tag));
